src/cascadia/TerminalControl/TermControl.xaml

```diff
@@ -1,4 +1,6 @@
-﻿<UserControl
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under
+the MIT License. See LICENSE in the project root for license information. -->

```

Comment:
*nit*: unrelated changes make git blame annoying. I'm not asking you to change this now, but try to make sure your changesets are constrained (within reason. Obviously I also say "while you're here, fix this" sometimes. It's a blurry line.)

Reply:
Ok, I'll at least try to keep these within the same file in the future. Would that work?

Reply:
I'd say... if you don't need to change files in a project, leave that project alone. When I generate changelogs for individual components, I will often do a `git log src/cascadia/WhateverComponent`, right? This would show me, when I do "git log TerminalCOntrol", that it changed to "represent inheritance in the settings UI". The cognitive load is higher for that, since I need to figure out why the TC had to change. Oh, it was just a copyright. _hrm._